woodwork/table_accessor.py

```diff
@@ -651,14 +662,15 @@ def pop(self, column_name):
 
         return series
 
-    def drop(self, columns):
+    def drop(self, columns, inplace=False):
         """Drop specified columns from a DataFrame.
 
         Args:
             columns (str or list[str]): Column name or names to drop. Must be present in the DataFrame.
+            inplace (bool): If False, return a copy. Otherwise, do operation inplace and return None.
```

Comment:
I think it would be good to mention that `inplace` is only supported when using pandas DataFrames.

Reply:
done!

Reply:
@tuethan1999 Did these changes get pushed? I'd think the docstring would mention something like "Only possible for pandas DataFrames."

Reply:
@tuethan1999 Looks like this update and the one below got missed. Can you please update the docstring to mention that `inplace` is only supported for pandas or something to that effect.

Reply:
Not sure what is happening, but the docstring here and for `rename` still don't seem to mention the limitation of only working with pandas dataframes.

Reply:
yeah, it seems like my changes didn't get pushed :/